source/renderer/app/actions/addresses-actions.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import Action from './lib/Action';
+
+// ======= ADDRESSES ACTIONS =======
+
+export default class AddressesActions {
+  createByronWalletAddress: Action<{
+    walletId: string,
+    isLegacy: boolean,
```

Comment:
@tomothespian we only deal with legacy wallets here - do we need this flag?

Reply:
@nikolaglumac I pass that but yeah, you are in right. I will remove isLegacy declaration in this flow